### PR TITLE
Add jambalya recipe

### PR DESCRIPTION
**Pull request by lancel0**

### CODE REVIEW

File: jambalya.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Jambalya
+
+## Ingredients
+
+**meat**
+
+* 1 1/4 lb - pork
+* 1 1/24 lb - smoked sausage
```
Comment (dgadling) [inline jambalya.md]: 1/24th of a lb?!

File: jambalya.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+* 1 tbsp - hot sauce
+
+**liquid**
+
+* 3 1/2 cup - water 
+* 2 cups - beef stock
+
+**optional**
+
+* 1 tsp - kitchen bouquet for color
```
Comment (dgadling) [inline jambalya.md]: What's "kitchen bouquet"?

File: jambalya.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+## How To
+
+The critical thing to get right here is the amount of liquid in the pot when the rice goes into cook.
+
+1.  Brown pork and remove from pot 
+2.  Brown sausage and remove from pot, drain most of the grease
+3.  Cook onions till caramelized
+4.  Add the rest of the vegetables and cook till tender
+5.  Add the meat and seasoning to the pot, add just enough water to the pot so you can stir.  Note this water is not from the 3 1/2 cups in the ingredients list.  Cook (simmer?) for around 30 minutes.  "You should be able to tell by the way it looks"
```
Comment (dgadling) [inline jambalya.md]: Anything besides "the way it looks" now that you've made it?

File: jambalya.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+6.  Mix beef broth and water; then add to pot
+7.  Bring pot to a boil 
+8.  Add rice and cook till most of the water is gone.  About 10 to 12 minute, stirring continuously on medium high heat.  The spoon should be able to lay on top of the jambalaya  without sinking.
+9.  Reduce to very low heat and cover pot.  DO NOT REMOVE cover for 15 minutes.
```
Comment (dgadling) [inline jambalya.md]: Want to make a note about the lowest setting you have?

File: jambalya.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+10. After 15 minutes remove cover and stir jambalaya.  The bottom of the pot may burn so carefully feel the bottom of the pot with the tip of spoon.  You do not want to stir the burned bits up.
```
Comment (dgadling) [inline jambalya.md]: Adding a note about the difference between "crispy" and "burnt" might be fun!

**Review by dgadling — CHANGES_REQUESTED** [review]
Looks like it needs a little help. Plus you've got experience making it now, so add that in!

File: jambalya.md (hunk 4 above)
Comment (standardtoaster) [reply jambalya.md]: There were also some kind of notes about the quality of the seal, and coping with a poor one, I think?

**Review by dgadling — APPROVED** [review]
Looks great! I'll make some cosmetic changes later

**Pull request merged**